Packs/ADPITAdmin/Integrations/ADP_IT_Admin/ADP_IT_Admin.yml

```diff
@@ -0,0 +1,72 @@
+category: IT Services
+commonfields:
+  id: ADP IT Admin
+  version: -1
+configuration:
+- additionalinfo: Client Id and Secret from ADP
+  defaultvalue: ""
+  display: Client Id
+  name: adp_credentials
+  required: true
+  type: 9
+- additionalinfo: Use this parameter if it's not configured in the XSOAR credentials
```

Comment:
I think a detailed description would help with the configuration because it's not very clear